### ISSUE TITLE
Tests for association proxies are failing

### ISSUE THREAD

**Issue by jfinkels** [jfinkels/flask-restless#480]
I cannot figure out how to implement and/or test many-to-many relationships via association proxies. I could really use the help of someone with more knowledge. I won't be able to release version 1.0.0 without these changes.

Start with, for example, `TestAssociationProxy` in `tests/test_fetching.py`.

**Comment by jfinkels** [comment]
I marked these tests to be skipped in commit 3ec4beaa68af05039e214d4fe6aa0c190142cbae so that we can use Travis correctly for pull requests, but the issues still remain.

**Comment by ademilsonfp** [comment]
+1

**Comment by jfinkels** [comment]
I have added tests and fixed some bugs to support association proxies in (the work-in-progress) pull request #608.

**Comment by azumafuji** [comment]
I started taking a look at this and I think I've made some progress.  It looks like the desired behaviour in `TestAssociationProxy` is to have the resultant data to include a `tags` attribute that is an array of the tag names.  The first issue is the default serialiser will not include the tags column because it's not a hybrid column (`serializers.py` line 289).  Is the desired behaviour when implementing this pattern to return the tags attribute and the list?  For example, when you implement a model with an attribute of type `association_proxy` that attribute should always be returned as a list by default?
